src/util/Constants.js

```diff
@@ -44,6 +45,7 @@ exports.DefaultOptions = {
   messageSweepInterval: 0,
   fetchAllMembers: false,
   disableMentions: 'none',
+  allowedMentions: {},
```

Comment:
Wouldn't be `undefined` be better here?
That way the option is also in code, but does not result in an additional property for every message, even if unused.

Reply:
I'm honestly unsure - I based it on the other defaults like `partials: [], presence: {}` but yeah, `undefined` is probably fine too.

Reply:
I'm in favor of `undefined` here since that way we would not end up sending `allowedMentions: {}` along _every_ message sent to any channel, but only if the value was explicitly provided in  either the client- or message options.